src/pages/City/City/City.tsx

```diff
@@ -93,7 +93,7 @@ const City = () => {
 
   const changeApproveStatus = async (memberId: number) => {
     setIsLoadingMemberId(memberId)
-    setIsLoadingPlus(false);  
+    setIsLoadingPlus(false)  
```

Comment:
What does it mean `setIsLoadingPlus`?

Reply:
It is useState. When we press the button "plus" that change value our state on false and turn on the button "loading"